File: Php/app/Cashgame/Details/ResultTable/ResultTableModel.php
namespace app\Cashgame\Details\ResultTable{
	use entities\Cashgame;
	use app\Cashgame\Details\ResultTableItem\ResultTableItemModel;
	use entities\Homegame;

	class ResultTableModel{

		/** @var ResultTableItemModel[] */
		public $resultModels;

		public function __construct(Homegame $homegame,
									Cashgame $cashgame){
			$results = getSortedResults($cashgame);
			$resultModels = array();
			foreach($results as $result){
				$resultModels[] = new ResultTableItemModel($homegame, $cashgame, $result);
			}
			resultModels = $resultModels;
		}

		private function getSortedResults(Cashgame $cashgame){
			$results = $cashgame.getResults();
			usort($results, 'entities\CashgameResultComparer::compareResult');
			return $results;
		}

	}

}

File: Web/Models/CashgameModels/Details/ResultTableModel.cs
using System.Collections.Generic;
using Core.Classes;
using System.Linq;

namespace Web.Models.CashgameModels.Details{

	public class ResultTableModel{

	    public List<ResultTableItemModel> ResultModels { get; set; }

		public ResultTableModel(Homegame homegame, Cashgame cashgame){
			var results = GetSortedResults(cashgame);
			var resultModels = new List<ResultTableItemModel>();
			foreach(var result in results){
				resultModels.Add(new ResultTableItemModel(homegame, cashgame, result));
			}
			ResultModels = resultModels;
		}

		private IEnumerable<CashgameResult> GetSortedResults(Cashgame cashgame)
		{
		    return cashgame.Results.OrderByDescending(o => o.Winnings);
		}

	}

}